components/clarinet-files/src/project_manifest.rs

```diff
@@ -383,12 +383,13 @@ fn get_epoch_and_clarity_version(
     settings_epoch: Option<&str>,
     settings_clarity_version: Option<&str>,
 ) -> Result<(StacksEpochId, ClarityVersion), String> {
-    // if neither epoch or version are specified in clarinet.toml use: epoch 2.05 and clarity 1
+    // if neither epoch or version are specified in clarinet.toml use: epoch DEFAULT_EPOCH and clarity 3
     // if epoch is specified but not version: use the default version for that epoch
 
     let epoch = match settings_epoch {
-        None => StacksEpochId::Epoch2_05,
+        None => DEFAULT_EPOCH,
```

Comment:
Needs to be `StacksEpochId::Epoch2_05`, otherwise it would be a breaking change for (really old) projects that didn't have the epoch set by default..
Maybe that would be an acceptable breaking change, but I'm not sure.
Maybe in future versions we'll just make the epoch setting required to have explicit settings and fully avoid this issue.

Reply:
That's reasonable. I should revert this since it's somewhat redundant with the "latest" addition anyway. No reason for a breaking change here